pkg/operator/controller/status/controller.go

```diff
@@ -8,6 +8,9 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
+	sailv1 "github.com/istio-ecosystem/sail-operator/api/v1"
+	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	gatewayapiv1 "sigs.k8s.io/gateway-api/apis/v1"
```

Comment:
nit: `gatewayapiv1 "sigs.k8s.io/gateway-api/apis/v1"` belongs with the other sigs.k8s.io imports.

Reply:
Historically, we've put controller-runtime imports in their own group.  I can move it to a separate group if you like.

Reply:
We've been ordering alphabetically by repo, that's why it belongs near or with the other sigs.k8s.io imports.

Reply:
Fixed in https://github.com/openshift/cluster-ingress-operator/compare/77beec1a8b6af8c8c5ce00f29f7a60683a84cd75..92e89a2a02fccb3d149e3e0491a30f19fe160774.